pkg/services/supportbundles/supportbundlesimpl/models.go

```diff
@@ -35,14 +35,19 @@ var (
 	}
 )
 
-func declareFixedRoles(ac accesscontrol.Service) error {
+func (s *Service) declareFixedRoles(ac accesscontrol.Service) error {
+	grants := []string{string(org.RoleAdmin), string(accesscontrol.RoleGrafanaAdmin)}
+	if s.serverAdminOnly {
+		grants = []string{string(accesscontrol.RoleGrafanaAdmin)}
```

Comment:
nit: I think `accesscontrol.RoleGrafanaAdmin` is already a string, so there's no need for casting it:
```suggestion
	grants := []string{string(org.RoleAdmin), accesscontrol.RoleGrafanaAdmin}
	if s.serverAdminOnly {
		grants = []string{accesscontrol.RoleGrafanaAdmin}
```